node/sc/config.go

```diff
@@ -91,10 +91,11 @@ type SCConfig struct {
 	AnchoringPeriod       uint64
 	SentChainTxsLimit     uint64
 
-	ParentChainID      uint64
-	VTRecovery         bool
-	VTRecoveryInterval uint64
-	Anchoring          bool
+	ParentChainID               uint64
+	VTRecovery                  bool
+	VTRecoveryInterval          uint64
+	Anchoring                   bool
+	ServiceChainDefaultGasLimit uint64
```

Comment:
Is this DefaultGasLimit or GasLimit? 
I understand that `1000000` is a default gas limit, but the configuration is gas limit, right?

Reply:
Yeah you are right. Changed field name from `ServiceChainDefaultGasLimit` to `ServiceChainOperatorGasLimit`.